tests/test_parser.py

```diff
@@ -23,7 +29,18 @@ def test_complex_outputs(sphinx_run, file_regression):
     # print(sphinx_run.status())
     assert sphinx_run.warnings() == ""
     assert sphinx_run.app.env.metadata == {
-        "complex_outputs": {"celltoolbar": "Edit Metadata", "hide_input": "False"}
+        "complex_outputs": {
+            "celltoolbar": "Edit Metadata",
+            "hide_input": "False",
+            "ipub": '{"bibliography": "example.bib", "biboptions": ["super", "sort"], "bibstyle": "unsrtnat", "language": "portuges", "listcode": true, "listfigures": true, "listtables": true, "titlepage": {"author": "Authors Name", "email": "authors@email.com", "institution": ["Institution1", "Institution2"], "logo": "logo_example.png", "subtitle": "Sub-Title", "supervisors": ["First Supervisor", "Second Supervisor"], "tagline": "A tagline for the report.", "title": "Main-Title"}, "toc": {"depth": 2}, "pandoc": {"use_numref": true, "at_notation": true}, "sphinx": {"bib_title": "My Bibliography"}}',
+            "jupytext": '{"metadata_filter": {"notebook": "ipub"}}',
+            "kernelspec": '{"display_name": "Python 3", "language": "python", "name": "python3"}',
+            "language_info": '{"codemirror_mode": {"name": "ipython", "version": 3}, "file_extension": ".py", "mimetype": "text/x-python", "name": "python", "nbconvert_exporter": "python", "pygments_lexer": "ipython3", "version": "3.6.2"}',
```

Comment:
@chrisjsewell is there any reason for us to think any of this stuff will be brittle and break over time? (e.g. as libraries update etc?)

Reply:
ugh I think the answer is yes, because the tests are erroring because the version of jupytext they've got is different from the one on my machine. hmmmmm